robot_driver/include/robot_driver/estimators/comp_filter_estimator.h

```diff
@@ -0,0 +1,82 @@
+#ifndef COMP_FILTER_H
+#define COMP_FILTER_H
+
+#include <robot_driver/estimators/state_estimator.h>
+
+//! Implements complementary filter as an estimator within the ROS framework.
+class CompFilterEstimator : public StateEstimator {
+ public:
+  /**
+   * @brief Constructor for CompFilterEstimator
+   * @return Constructed object of type CompFilterEstimator
+   */
+  CompFilterEstimator();
+
+  /**
+   * @brief initialize Complementary Filter
```

Comment:
This is very pedantic, but I think the Doxygen comments will look much cleaner if the first word of a comment is capitalized (apply elsewhere in PR).